stylesheets/styles.css

```diff
@@ -331,13 +337,6 @@ pr-badge a:focus::after  {
 }
 
 /* Min-width break to add more padding on wider screens */
-@media all and (max-width: 720px) {
-	#sidebar {
-		display: none;
-	}
-}
-
-/* Min-width break to add more padding on wider screens */
```

Comment:
Class `closed` is visible on larger screens, check [L358-361](https://github.com/woubuc/1pr/blob/f7c6754395a05d762de2718017f4afce2bb7432b/stylesheets/styles.css#L358-L361)